pygsheets/cell.py

```diff
@@ -278,7 +296,7 @@ def get_json(self):
             nformat, pattern = self.format, ""
         return {"userEnteredFormat": {
                         "numberFormat": {
-                            "type": nformat.value,
```

Comment:
Don't remove it, will throw an error as `nformat` is an enum and we want the actual text